### ISSUE TITLE
Range slider: Screen reader callouts lack context

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5430]
# Summary

When using our range slider with a screen readers, users are not told the min, max, or any other context to let users know where there are within the screen reader.

From @amycole501 in #5413

>This sounds good in JAWS. I can hear "10, 20, 30" etc, and can use the arrow keys to move up and down the range slider. I recommend that whatever the range is indicating be written into the help text. For example "ten percent" instead of just "ten" if the slider is indicating a percentage range.
>
>In NVDA the action is identical to JAWS. I used Chrome to test both.

_Originally posted by @amycole501 in https://github.com/uswds/uswds/pull/5413#pullrequestreview-1563505343_

**Comment by mahoneycm** [comment]
Duplicate of #5431 - Closing
